docs/root/configuration/http/http_filters/aws_lambda_filter.rst

```diff
@@ -20,11 +20,66 @@ If :ref:`payload_passthrough <envoy_api_msg_config.filter.http.aws_lambda.v2alph
 *Note*: This means you lose access to all the HTTP headers in the Lambda function.
 
 However, if :ref:`payload_passthrough <envoy_api_msg_config.filter.http.aws_lambda.v2alpha.config>`
-is set to ``false``, then the HTTP request is transformed to a JSON (the details of the JSON transformation will be
-documented once that feature is implemented).
+is set to ``false``, then the HTTP request is transformed to a JSON payload with the following schema:
+
+.. code-block::
+
+    {
+        "path": "/path/to/resource",
+        "httpMethod": "GET|POST|HEAD|...",
```

Comment:
Isn't it redundant to say `httpMethod`? This whole model looks very http-centric right now.

Makes me wonder about how other protocol translations could work in the future...

Reply:
are you suggesting to rename it to "method"? or to remove it completely?

Reply:
I meant renaming it method.